src/lib/swapper/constants.ts

```diff
@@ -11,7 +11,7 @@ import { thorchainSwapper } from 'lib/swapper/swappers/ThorchainSwapper/Thorchai
 import { zrxApi } from 'lib/swapper/swappers/ZrxSwapper/endpoints'
 import { zrxSwapper } from 'lib/swapper/swappers/ZrxSwapper/ZrxSwapper'
 
-export const QUOTE_TIMEOUT_MS = 10_000
+export const QUOTE_TIMEOUT_MS = 60_000
```

Comment:
suggest we leave this in while nodes are struggling - not a lifi issue, its an us thing (unchained gas estimation for BNB very slow)